swagger-codegen/2.1.5.wso2v1/pom.xml

```diff
@@ -0,0 +1,132 @@
+                            nodejs.*;version="${export.pkg.version.swagger-codegen}",
+                            JMeter.*;version="${export.pkg.version.swagger-codegen}",
+                            JavaSpringMVC.*;version="${export.pkg.version.swagger-codegen}",
+                            Javascript.*;version="${export.pkg.version.swagger-codegen}",
+                            JavaJaxRS.*;version="${export.pkg.version.swagger-codegen}",
+                            JavaInflector.*;version="${export.pkg.version.swagger-codegen}",
+                            htmlDocs.*;version="${export.pkg.version.swagger-codegen}",
+                            Groovy.*;version="${export.pkg.version.swagger-codegen}",
+                            flaskConnexion.*;version="${export.pkg.version.swagger-codegen}",
+                            flash.*;version="${export.pkg.version.swagger-codegen}",
+                            dart.*;version="${export.pkg.version.swagger-codegen}",
+                            CsharpDotNet2.*;version="${export.pkg.version.swagger-codegen}",
+                            csharp.*;version="${export.pkg.version.swagger-codegen}",
+                            config.*;version="${export.pkg.version.swagger-codegen}",
+                            codegen.*;version="${export.pkg.version.swagger-codegen}",
+                            clojure.*;version="${export.pkg.version.swagger-codegen}",
+                            asyncscala.*;version="${export.pkg.version.swagger-codegen}",
+                            android.*;version="${export.pkg.version.swagger-codegen}",
+                            akka-scala.*;version="${export.pkg.version.swagger-codegen}",
+                            Java.*;version="${export.pkg.version.swagger-codegen}",
+                            io.swagger.*;version="${export.pkg.version.swagger-codegen}"
+                        </Export-Package>
+                        <Import-Package>
+                            org.apache.commons.io;version="1.4",
+                            org.apache.commons.io.comparator;version="1.4",
+                            org.apache.commons.io.filefilter;version="1.4",
+                            org.apache.commons.io.input;version="1.4",
+                            org.apache.commons.io.output;version="1.4",
+                            org.joda.time.*;version="2.9.2"
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Include-Resource>
+                            {maven-resources}
+                        </Include-Resource>
+                        <Embed-Dependency>
+                            joda-time;scope=compile|runtime;inline=false,
+                            commons-io;scope=compile|runtime;inline=false
+                        </Embed-Dependency>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
This library dynamically loades some classes using java.lang.Class.forName() method and that's whyI have used DynamicaImports.